test/cyclic_model.xml

```diff
@@ -0,0 +1,36 @@
+<model version="2.1">
+  <elements>
+    <e n="cyclic" t="dir" architecture_modularity="68" author_count_1="1" author_count_180="1" author_count_30="1" author_count_365="1" author_count_7="1" author_count_90="1" author_list_1="NotMixu &lt;mikael@softagram.com&gt;" author_list_180="NotMixu &lt;mikael@softagram.com&gt;" author_list_30="NotMixu &lt;mikael@softagram.com&gt;" author_list_365="NotMixu &lt;mikael@softagram.com&gt;" author_list_7="NotMixu &lt;mikael@softagram.com&gt;" author_list_90="NotMixu &lt;mikael@softagram.com&gt;" bug_count_1="" bug_count_180="" bug_count_30="" bug_count_365="" bug_count_7="" bug_count_90="" bug_fix_commit_count_1="0" bug_fix_commit_count_180="0" bug_fix_commit_count_30="0" bug_fix_commit_count_365="0" bug_fix_commit_count_7="0" bug_fix_commit_count_90="0" bug_fix_ratio_1="" bug_fix_ratio_180="" bug_fix_ratio_30="" bug_fix_ratio_365="" bug_fix_ratio_7="" bug_fix_ratio_90="" bug_list_1="" bug_list_180="" bug_list_30="" bug_list_365="" bug_list_7="" bug_list_90="" commit_count_1="4" commit_count_180="4" commit_count_30="4" commit_count_365="4" commit_count_7="4" commit_count_90="4" compilers="" days_since_modified="0" eliminated_files="" feature_count_1="" feature_count_180="" feature_count_30="" feature_count_365="" feature_count_7="" feature_count_90="" feature_list_1="" feature_list_180="" feature_list_30="" feature_list_365="" feature_list_7="" feature_list_90="" last_modified="2023-02-15 15:59 +0200" latest_commits="8642b5f - 2023-02-15 15:59 +0200 - mikael@softagram.com - Hmm;203f2e3 - 2023-02-15 15:52 +0200 - mikael@softagram.com - Hmm;6193e95 - 2023-02-15 13:44 +0200 - mikael@softagram.com - Change to init analysis;f441ce7 - 2023-02-15 13:41 +0200 - mikael@softagram.com - Init" loc_blank="0" loc_comment="0" loc_without_comments_and_testcode="0" risk_density="84.0" risk_level="11.4" risk_points="228" softagram_index="" softagram_version="version" target_label="N/A" tech_debt_1="" tech_debt_180="" tech_debt_30="" tech_debt_365="" tech_debt_7="" tech_debt_90="" testcode_loc="20">
```

Comment:
You could clean up this test model file to include only data which is relevant for the test. Remove unneeded attributes and especially email addresses, since it is personal data.